README.md

```diff
@@ -2,7 +2,9 @@ AsseticBundle
 =============
 
 The `AsseticBundle` provides integration of the [Assetic](https://github.com/kriswallsmith/assetic)
-library into the Symfony2 framework.
+library into the Symfony framework (up to Symfony 3.4).
+
+As of Symfony 4.0, Symfony applications should use Webpack Encore. 
```

Comment:
Maybe prefix this with `**CAUTION**:` to put some emphasize on this paragraph?

Reply:
Good idea, done.